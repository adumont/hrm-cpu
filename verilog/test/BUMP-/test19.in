42 5f
27 f9
25 3c
32 a1
52 5b
25 83
23 da
21 24
04 30
51 19
53 80
28 29
30 b8
24 b9
15 4c
50 1d
59 dc
21 f2
27 f4
31 91
11 ef
23 ba
04 dd
45 8b
19 3d
01 65
43 1d
