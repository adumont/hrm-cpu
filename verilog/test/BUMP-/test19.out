5e
f8
3b
a0
5a
82
d9
23
2f
18
7f
28
b7
b8
4b
1c
db
f1
f3
90
ee
b9
dc
8a
3c
64
1c
